ownership-chain/runtime/src/xcm_config.rs

```diff
@@ -189,6 +190,26 @@ pub type OnOwnershipParachainBlobDispatcher =
 /// XCM weigher type.
 pub type XcmWeigher = FixedWeightBounds<UnitWeightCost, RuntimeCall, MaxInstructions>;
 
+/// Logic for sending fees to the sudo account. On every unbalanced change, the amount is
```

Comment:
this sends transaction fees from xcm operations to sudo account. in the future, when we have the treasury, we can change this impl. but for now, I needed some account to send the fees, so I am using sudo